frontend/app/performance/lighthouse/LighthouseOrchestrator.mjs

```diff
@@ -171,18 +173,17 @@ export class LighthouseOrchestrator {
    */
   async runLighthouse(appName, mode, runId) {
     const sanitizedRunId = path.basename(runId)
-    console.log(`Running Lighthouse for ${appName} in ${mode} mode`)
+    const appNameRef = (appName.split("/").pop() || "")?.split(".")[0]
```

Comment:
nit: what's meant with `Ref` here? Based on the code it looks like the top-level name or similar, maybe that's a better name?

Reply:
I'll rename this to `appBaseName`, since it's the name of the app without any of the parent directories and/or file extension.